CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ Internal:
 (PR [#574](https://github.com/alphagov/govuk-frontend/pull/574))
 - Ensure render function does not have undefined object
   (PR [#587](https://github.com/alphagov/govuk-frontend/pull/587))
+- Fix components relying on global builds
```

Comment:
Given footer hasn't been released yet, is this something we need to document? As the thing we're fixing was never actually released?

Reply:
Agreed, that's why this change is under 'Internal' not 'Fix'